slack-base/src/main/java/com/hubspot/slack/client/models/events/util/AbstractSlackReplyMessage.java

```diff
@@ -26,10 +30,27 @@
   @JsonProperty("channel")
   public abstract String getChannelId();
 
-  public abstract List<Reply> getReplies();
```

Comment:
Slack docs don't mention if the update applies to the [message_replied](https://api.slack.com/events/message/message_replied) event. I reached out to them and they said

> ... our team will be doing some testing to confirm this for you and will send you an update...

If they don't update the `message_replied` event, we won't need to (and shouldn't) update this model

Reply:
How did you reach out to them? I don't see an email or anything in our hubspot-slack Workspace.

Reply:
I just [opened a support ticket](slack.com/help/requests/new). I got a response from them:
```
Hi there,

Thanks for getting in touch. The message_replied event subtype will not be affected by the changelog entry that you mentioned. The message_replied subtype simply indicates that a new message has been posted, and it's a "reply" to another message.

That event will not contain other threaded replies, so you would need to use the conversations.replies API method to fetch those: https://api.slack.com/methods/conversations.replies.
```

So it looks like this file doesn't need to be modified

Reply:
sounds good. I'll revert that.

Reply:
Slack came back and said all events are affected as well, so asked them again to clarify based on this specific example. Given what they come back with, I'll update this to have a default of empty array too
https://hubspot.slack.com/help/requests/2419401